src/sql/azdata.proposed.d.ts

```diff
@@ -421,6 +421,20 @@ declare module 'azdata' {
 		defaultValueOsOverrides?: DefaultValueOsOverride[];
 	}
 
+	/**
+	 * The connection string options for connection provider.
+	 */
+	export interface ConnectionStringOptions {
```

Comment:
Why is this in the extension API? It's just a package.json thing which typically doesn't go here (VS Code has JSON validation for this, at some point we should probably look into whether we can make our own version of that which has support for validating our package.json entries)

Reply:
agreed, removed it from typings file.